### ISSUE TITLE
Remove ontobio's usage of deprecated networkx methods

### ISSUE THREAD

**Issue by deepakunni3** [biolink/ontobio#398]
Networkx 2.4 has deprecated a few methods including `graph.node[]`. The recommendation is to use `graph.nodes[]`.

All calls to `graph.node` --> `graph.nodes`

**Comment by matentzn** [comment]
Yes please.. In general, it would be great if we could update to a newer networkx version because of all the breaking changes between 2.2 and 2.3.

@jannahastings also had trouble with that, when trying the library
